Library/Homebrew/hardware.rb

```diff
@@ -55,6 +55,8 @@ def self.cores_as_words
     when 1 then "single"
     when 2 then "dual"
     when 4 then "quad"
+    when 6 then "hexa"
```

Comment:
by non-europeans I more meant "people who haven't come across much Latin/Greek" which may or may not apply to @ilovezfs 😀